docs/quickstart.md

```diff
@@ -74,11 +74,11 @@ So let's get started.
 
 ## Step 1: Create a cluster
 
-Choose an `$OPSTRACE_CLUSTER_NAME` for your cluster (it must be 13 characters or less):
+Choose an `OPSTRACE_CLUSTER_NAME` for your cluster (it must be 13 characters or less):
 
 <!--input:$OPSTRACE_CLUSTER_NAME-->
 
-The cluster name will globally identify you in the provided `opstrace.io` domain, which we provide for you by default.
+The cluster name will globally identify you `$OPSTRACE_CLUSTER_NAME.opstrace.io` domain, which we provide for you by default.
```

Comment:
@triclambert the patch for this line was bad; you on it?

Reply:
Thanks for catching that.  Put up #99 for you to review.